### PR TITLE
Update code with latest version. Make end2end notebook.

### PR DESCRIPTION
**Pull request by veseln**
Changes made: 
* Code updated to latest internal version.
*  Removed separate notebooks and replaced it with singular end2end notebook. 
* Moved scripts to `fd/` folder. Adapted the scripts so they can be both imported and run as scripts. 

Still relies on eo-learn 0.10.2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,33 +74,33 @@ This repository has the following content:
 
  * `fd`: modules implementing each part of the workflow;
  * `input-data`: folder storing the file defining the AOI and the consequent grid definition file;
- * `notebooks`: folder storing the ordered notebooks to execute the end-to-end workflow.
+ * `notebooks`: folder storing the examplesmple  notebook to execute the end-to-end workflow.
```
Comment (devisperessutti) [inline README.md]: typo in example

File: fd/prediction.py [hunk 2]
```diff
@@ -10,26 +9,20 @@
...
-
-from eolearn.core import EOPatch, FeatureType, LoadTask, SaveTask, OverwritePermission
-from eoflow.models.segmentation_unets import ResUnetA
-
 from eoflow.models.losses import TanimotoDistanceLoss
-
 from eoflow.models.metrics import MCCMetric
+from eoflow.models.segmentation_unets import ResUnetA
+from eolearn.core import EOPatch, FeatureType, LoadTask, OverwritePermission, SaveTask
+from fs.copy import copy_dir, copy_file
```
Comment (devisperessutti) [inline fd/prediction.py:21]: why are we changing the imports?

File: fd/scripts/download.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
+#
```
Comment (devisperessutti) [inline fd/scripts/download.py:1]: general comment to the scripts below, should we introduce a running number to the filename (as we had for notebooks), to make clearer in which order they should be executed?

**Review by devisperessutti — CHANGES_REQUESTED** [review]
minor comments

File: fd/prediction.py (hunk 2 above)
Comment (veseln) [reply fd/prediction.py:21]: Mah, I would assume there were some formatting differences between the version here and our latest internal version. These changes I didn't make but were made when updating to the latest version...

File: fd/scripts/download.py (hunk 3 above)
Comment (veseln) [reply fd/scripts/download.py:1]: I think if we want the scripts to be used within the notebook we can't :speak_no_evil:  
I.e, trying to import: 
`from fd.scripts.01_download import batch_download`
gives: 
`SyntaxError: invalid decimal literal`

The current solution is a bit hacky/ugly currently. All the scripts have the same form with two parts: 
* argument parsing 
* the core function that actually does stuff

The core functions could be moved into separate file(s) (which could be imported) and have the scripts just be the argument parsing and the calling of the core function (these could be named whatever as they wouldn't be importable modules)

Not sure if this is something we want to spend time on tho. The order is pretty clear both in the documentation and in the end2end notebook.

**Comment by devisperessutti** [comment]
Notebook `field-delineation-end2end` looks great.

Minor comments:

 * third cell of text, I would change **The crop ground truth data** to **Field boundary ground-truth data required for training**
 * text in Setup credentials "A bucket for demonstration purposes is provided by Sinergise." -> Nope :smile: -> delete
 * do we define a TIME_INTERVAL variable, as it is likely to be changed by users ?
 * in **Create .npz files from patchlets** section, missing space in "This dataframe is later used forthe cross-validation splits. "

File: fd/scripts/download.py (hunk 3 above)
Comment (devisperessutti) [reply fd/scripts/download.py:1]: ahh yes, forgot about the imports. Let's leave it as is, the order is indeed clear from docs and notebook

File: fd/prediction.py (hunk 2 above)
Comment (devisperessutti) [reply fd/prediction.py:21]: OK, just curious about what happened, not an issue IMO

**Pull request merged**